source/common/router/router.h

```diff
@@ -364,10 +364,11 @@ class Filter : Logger::Loggable<Logger::Id::router>,
                public RouterFilterInterface {
 public:
   Filter(FilterConfig& config, FilterStats& stats)
-      : config_(config), stats_(stats), downstream_1xx_headers_encoded_(false),
-        downstream_response_started_(false), downstream_end_stream_(false), is_retry_(false),
-        request_buffer_overflowed_(false), streaming_shadows_(Runtime::runtimeFeatureEnabled(
-                                               "envoy.reloadable_features.streaming_shadow")) {}
+      : config_(config), stats_(stats), grpc_request_{}, exclude_http_code_stats_(false),
```

Comment:
nit: to make it easier to read and consistent with other fields in this MIL
```suggestion
      : config_(config), stats_(stats), grpc_request_(false), exclude_http_code_stats_(false),
```